L/libcvrpsep/build_tarballs.jl

```diff
@@ -0,0 +1,37 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libcvrpsep"
+version = v"0.0.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/chkwon/CVRPSEP.git", "c4d1720ffafcc174e39f65c7a90146cb82b68e44")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd CVRPSEP/
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
```

Comment:
This should help with the failure:
```suggestion
autoreconf -fiv
./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
```